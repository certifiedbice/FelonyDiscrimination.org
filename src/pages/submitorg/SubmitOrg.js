import React from 'react';
import SubmitOrgForm from '../../components/submitorgform/SubmitOrgForm';
import './SubmitOrg.css';

export default function SubmitOrg(){return(<SubmitOrgForm/>);}