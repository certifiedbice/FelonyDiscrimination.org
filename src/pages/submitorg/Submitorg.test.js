import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import {BrowserRouter} from 'react-router-dom';
import SubmitOrg from './SubmitOrg';

it('renders without crashing',()=>{
    const div=document.createElement('div');
    ReactDOM.render(<BrowserRouter><SubmitOrg/></BrowserRouter>,div);
});

it('renders correctly',()=>{
    const tree=renderer
        .create(<BrowserRouter><SubmitOrg/></BrowserRouter>)
        .toJSON();
        expect(tree).toMatchSnapshot();
});