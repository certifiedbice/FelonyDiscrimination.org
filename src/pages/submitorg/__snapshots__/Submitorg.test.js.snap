// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  id="form-container"
>
  <div
    id="endorsement-login-message"
  >
    <h3>
      You must 
      <a
        href="/login"
        onClick={[Function]}
      >
        log in
      </a>
       to submit organizations
    </h3>
  </div>
</section>
`;
